docs/modules/spring/pages/overview.adoc

```diff
@@ -1,7 +1,33 @@
 = Integrate with Spring
 
-You can integrate Hazelcast with Spring and this section explains the
-configuration of Hazelcast within Spring context.
+https://spring.io/[Spring] is a popular framework for writing enterprise-grade Java applications. https://spring.io/projects/spring-framework[Spring Framework] includes a range of libraries to reduce the need for boilerplate code and encourages a consistent structure for ease of maintenance. https://spring.io/projects/spring-boot[Spring Boot] further simplifies development by automatically configuring dependencies based on your classpath.
 
-Supported Versions are Spring 4.3 and higher releases and the latest
-tested Spring version is 5.3.
+Hazelcast Platform is integrated with the Spring Framework and Spring Boot, allowing you to easily incorporate Hazelcast's fast data cache capabilities into your Spring applications.
+
+== Why use Hazelcast with Spring?
+
+By integrating Hazelcast into your Spring application, you can:
+
+* Initialize a Hazelcast distributed map (IMap) object without needing to configure a Hazelcast member first. Spring Boot automatically configures a Hazelcast member if Hazelcast is on the classpath.
+* Use `@SpringAware` annotation on Hazelcast objects to enable dependency injection.
+* Reduce the configuration needed to set up Hazelcast.
+
+Hazelcast can be used as a Spring cache manager or a JCache-compliant cache, as described in the https://docs.spring.io/spring-boot/reference/io/caching.html#io.caching.provider.hazelcast[Spring Boot documentation].
+
+== Architecture options
+
+You can embed Hazelcast members in the same Java process as your Spring application or deploy them separately in a client/server topology. See xref:deploy:choosing-a-deployment-option.adoc[].
+
+== Supported versions
+// TODO: update
+**Spring Framework**
+
+Supported: 
+
+Latest tested: 
+
+**Spring Boot**
+
+Supported:
```

Comment:
All Spring Boot-related code is inside Spring's codebase. We don't support any concrete versions. We can link to https://docs.spring.io/spring-boot/reference/io/hazelcast.html